src/NuGet.Clients/NuGet.PackageManagement.UI/Utility/PackageSourceMappingUtility.cs

```diff
@@ -91,5 +91,52 @@ internal static class PackageSourceMappingUtility
 
             mappingProvider.SavePackageSourceMappings(newAndExistingPackageSourceMappingItems);
         }
+
+        internal static string? GetNewSourceMappingSourceName(PackageSourceMapping packageSourceMapping, PackageSourceMoniker activePackageSourceMoniker)
+        {
+            string? sourceMappingSourceName = packageSourceMapping.IsEnabled
+                && !activePackageSourceMoniker.IsAggregateSource
+                ? activePackageSourceMoniker.PackageSourceNames.First() : null;
```

Comment:
NIT: the indentation looks off. I'm curious what the C# style analyzer would recommend. Maybe we should consider start using it for our codebase. Anyway, my suggestion is: 

string? sourceMappingSourceName = packageSourceMapping.IsEnabled 
    && !activePackageSourceMoniker.IsAggregateSource ?            
               activePackageSourceMoniker.PackageSourceNames.First() : null;

Reply:
According to our coding guideline, we indent with 4 spaces.
`                                      &&` doesn't seem right to me. 
I'm not sure which C# analyzer would be most applicable, but the `case` statement also uses 4 spaces:
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/csharp-formatting-options#csharp_indent_case_contents

Maybe others can clarify what the team prefers.

Reply:
We already have whitespace analyzers, but they can't catch stuff like that. 
The indentation is correct. An extra level of indentation for certain types is not something we have configured. not sure if we can.

Reply:
There are two parts to my comment. One is starting the indentation for a line with an assignment after the = operator. The other is not started a line with the ? operator and instead having the ? operator at the end of the previous line. The latter is more important IMO. Nikolche has a couple of suggestions that he'll add here.

Reply:
Personally, I like having the operators at the end. IMO it makes it more readable.

Example: 
```cs
string? sourceMappingSourceName = packageSourceMapping.IsEnabled && 
     !activePackageSourceMoniker.IsAggregateSource ?
          activePackageSourceMoniker.PackageSourceNames.First() : null;
```
Or

```cs
string? sourceMappingSourceName = packageSourceMapping.IsEnabled && 
     !activePackageSourceMoniker.IsAggregateSource ?
          activePackageSourceMoniker.PackageSourceNames.First() : 
          null;
```

I feel like given that `&&` and `?, : ` operators are of different priority, they should be intended differently. 

It's be great if we can configure the style analyzers to enforce it.